nautobot_chatops/dispatchers/adaptive_cards.py

```diff
@@ -121,7 +122,13 @@ def multi_input_dialog(self, command, sub_command, dialog_title, dialog_list):
             ],
         }
         blocks.append(buttons)
-        return self.send_blocks(blocks, callback_id=callback_id, modal=True, ephemeral=False, title=dialog_title)
+        return self.send_blocks(
+            blocks,
+            callback_id=callback_id,
+            modal=True,
+            ephemeral=settings.PLUGINS_CONFIG["nautobot_chatops"]["send_all_messages_private"],
```

Comment:
Can an adaptive-cards multi-input dialog even be sent as ephemeral=True? I *think* dialogs are always private to a specific user, but would want to see this tested.

Reply:
This is a regression of the last time on it too. I had some challenges on my dev environment and missed getting this cleared up. I will remove the ephemeral part on this.